src/js/components/tabs.js

```diff
@@ -0,0 +1,26 @@
+import { renderPage, changeTabsPage } from '../util'
+
+const createTabs = (id, items) => {
+  const tabs = document.createElement('div');
+  tabs.setAttribute('id', id);
+  tabs.setAttribute('class', 'tabs');
+  const tabsLinks = document.createElement('div');
+  tabsLinks.setAttribute('class', 'tabs-items');
+  for(let i = 0; i < items.length; i++) {
```

Comment:
I would suggest it's better to use `forEach` here :)

Reply:
I prefer to use for loop if I need to use indexes.